hoot-core/src/main/cpp/hoot/core/algorithms/Soundex.cpp

```diff
@@ -39,7 +39,7 @@
 namespace hoot
 {
 
-HOOT_FACTORY_REGISTER(StringDistance, Soundex);
+HOOT_FACTORY_REGISTER(StringDistance, Soundex)
```

Comment:
Ah, yes, the extra semicolon.  When in doubt add a semicolon!  Not too many of these but when I see them, I remove them.